### PR TITLE
Feature/ansibleintegration

### PR DESCRIPTION
**Pull request by joerivrij**

### CODE REVIEW

File: create_docker_compose.py [hunk 1]
```diff
@@ -111,9 +111,11 @@
 # For production environment
 if PROFILE == 'prod':
     dataseeder_arg.default = False
-    image_action_arg.default = 'build'
-    tag_action_arg.default = os.environ.get('CIRCLE_SHA1')
+    image_action_arg.default = 'pull'
+    tag_action_arg.default = 'production'
+    exclude_jaeger_arg.default = True
```
Comment (mrck) [inline create_docker_compose.py]:
```suggestion
    # We exclude jaeger for now, as the 1 GB droplet can not handle the database requirements
    exclude_jaeger_arg.default = True
```

File: deployment/ansible/ansible.cfg [hunk 2]
```diff
@@ -1,4 +1,4 @@
 [defaults]
-hostfile = ./hosts
-private_key_file = ~/.ssh/id_rsa
-host_key_checking = False
\ No newline at end of file
+host_key_checking = False
+[ssh_connection]
+retries = 5
```
Comment (mrck) [inline deployment/ansible/ansible.cfg]:
```suggestion
retries = 5

```

File: deployment/ansible/droplet_provision_centos.yml [hunk 3]
```diff
@@ -86,14 +75,23 @@
...
+    mount:
+      path: /mnt/datavolumedocker
+      src: /dev/disk/by-id/scsi-0DO_Volume_datavolumedocker
+      opts: discard,defaults,noatime
+      fstype: ext4
+      state: mounted
+    ignore_errors: no
+
+  - command: "docker-compose -f /home/terraformuser/docker/docker-compose-created.yml up -d"
     register: compose_up
```
Comment (mrck) [inline deployment/ansible/droplet_provision_centos.yml]: - [ ] Restart SSH service, to block future terraformuser logins

Comment (joerivrij) [reply deployment/ansible/droplet_provision_centos.yml]: Done

File: deployment/ansible/droplet_provision_centos.yml [hunk 4]
```diff
@@ -86,14 +75,29 @@
...
+    notify:
+    - restart sshd
+
+  - debug: msg="{{ compose_up }}"
 
-  - debug: msg="{{ compose_up }}"
\ No newline at end of file
+  handlers:
+    - name: restart sshd
+      service: name=sshd state=restarted
```
Comment (mrck) [inline deployment/ansible/droplet_provision_centos.yml]:
```suggestion
      service: name=sshd state=restarted
      
```

**Pull request merged**